composer.json

```diff
@@ -41,9 +41,9 @@
         }
     ],
     "require": {
-        "php": "^8.1",
+        "php": ">=8.1",
         "dragon-code/support": "^6.10",
-        "symfony/console": "^5.3 || ^6.0"
+        "symfony/console": "^5.3 || >=6.0"
```

Comment:
In this case, you should not use versioning with `>=`, as it will not only include Symfony version 6, but also 7, 8, and so on.

It is better to add tests for version 7 and specify it explicitly. This way we will make sure that the package works with version 7.